static/css/ProfilePage.css

```diff
@@ -125,4 +117,35 @@ main{
     background-color: white;
     border: 0;
     border-radius: 1em 1em 1em 1em;
+}
```

Comment:
Use our color scheme for the modal in variables similarly to how we did it on the other css pages. A simple copy and paste of those variables and then using them in according places would work. And would make the modal look better